### PR TITLE
docs: add contributing. #53

### LINKED ISSUE #28: link to roadmap from readme
It would be useful if the readme or another public place linked to the roadmap

### PR DESCRIPTION
**Pull request by juliangruber**
Closes #28

### CODE REVIEW

**Review by bajtos — COMMENTED** [review]
@juliangruber could you please format the markdown using `prettier`?

We don't have a CI check to run prettier yet, but the configuration files are already in place - see e.g. [.prettierrc](https://github.com/filecoin-station/zinnia/blob/main/.prettierrc.yaml)

File: README.md [hunk 1]
```diff
@@ -40,3 +40,12 @@ computers.
 ## Basic use
 
 See [CLI](./cli/README.md).
+
+## Contributing
+
+The [roadmap](https://www.notion.so/pl-strflt/Zinnia-Roadmap-b849d322502f4c079525cd8cc887df02) and [issue tracker](https://github.com/filecoin-station/zinnia/issues) are great places to check out for your first
```
Comment (bajtos) [inline README.md:46]: I think this is effectively closing the following issue:
- https://github.com/filecoin-station/zinnia/issues/28

**Comment by bajtos** [comment]
> We don't have a CI check to run prettier yet, but the configuration files are already in place - see e.g. [.prettierrc](https://github.com/filecoin-station/zinnia/blob/main/.prettierrc.yaml)

See
- #22

File: README.md (hunk 1 above)
Comment (juliangruber) [reply README.md:46]: good catch!

**Comment by juliangruber** [comment]
I ran prettier, was it right like this?

File: runtime/vendored/asserts.bundle.js [hunk 2]
```diff
@@ -2,786 +2,912 @@
 // deno-lint-ignore-file
 // This code was bundled using `deno bundle` and it's not recommended to edit it manually
 
-const { Deno  } = globalThis;
+const { Deno } = globalThis;
```
Comment (bajtos) [inline runtime/vendored/asserts.bundle.js:5]: We don't want to (re)format vendored files. Could you please update [.prettierignore](https://github.com/filecoin-station/zinnia/blob/main/.prettierignore) to skip everything in `runtime/vendored`?

Either in this PR, or else:
1. Revert changes in this file, land the rest of this PR.
2. Open a new PR to fix prettier ignore config.

Comment (bajtos) [reply runtime/vendored/asserts.bundle.js:5]: > I ran prettier, was it right like this?
> `npx prettier -w .`

Yes, that looks good to me!

**Review by bajtos — CHANGES_REQUESTED** [review]
> I ran prettier, was it right like this?
> `npx prettier -w .`

Yes, that looks good to me!

Except we don't want to (re)format vendored files, see my comment below.

**Review by bajtos — APPROVED** [review]

**Pull request merged**